docs/building-a-form/common-definitions.md

```diff
@@ -45,22 +47,25 @@ Available definitions are:
 ### Autosuggest
 
 A common type-ahead widget that lets a user type in values and narrow down a longer list of options. It is most commonly used with an `enum` of the available options as shown here. Define the uiSchema by calling the function that you import. You can pass an object with additional uiSchema properties.
+
 ```js
 import { uiSchema as autosuggestUI } from 'us-forms-system/lib/js/definitions/autosuggest';
 
-schema: {
-  type: 'object',
-  properties: {
-    officeLocation: {
-      type: 'string',
-      enum: [
-        'LA', 'NY', 'CH'
-      ],
-      enumNames: [
-        'Los Angeles',
-        'New York',
-        'Chicago'
-      ]
+function schema() {
```

Comment:
I think what these code samples are trying to show is what this would look like in the form config, not what the definition code is. So I think this would actually remain the way it was before, where the `schema` is an object in the form config. You can just revert this change for the `schema`.

This particular definition is a little confusing because here we're not actually using the `schema` from the definition, but instead writing our own schema into the form config, so I think we should clean that up as a separate issue. I'll create a ticket for that.